src/web/server.ts

```diff
@@ -209,6 +210,11 @@ export async function startServer(
     res.json({ data: await getTestCases() });
   });
 
+  app.get('/api/evals/:datasetId', async (req, res) => {
```

Comment:
This route would make more sense as `/api/evals?datasetId=...`

Reply:
updated